README.md

```diff
@@ -1,2 +1,20 @@
 # dapp-psm
+
 UI for Inter Protocol PSM
+
+## Development
+
+`yarn dev` to start a local HMR server.
+
+By default it it will connect to the prod Wallet UI. To connect to a local one, use
+http://127.0.0.1:5173/?wallet=local .
```

Comment:
It also works with different deployments of `wallet-app` for testing. I.e. `https://<branch-name>.wallet-app.pages.dev` can be used with `?wallet=<branch-name>`

Reply:
yeah. I figured that's advanced mode. Feel free to amend the commit or PR to include